### ISSUE TITLE
Install JSONStat like a package

### ISSUE THREAD

**Issue by Jonathan7SSB** [klpn/JSONStat.jl#1]
I cant see any descriptions for how one can install this package, and if one tries as normal installation in Julia REPL i only get Error that it cant fint package or project file in package. If everything here works well, then the code/functions are of interest, and JSON-stat themself are refering to this package when it comes to Julia: https://json-stat.org/tools/#top

**Comment by klpn** [comment]
This is, as you can see, an old package, which has not been updated for modern Julia packaging systems, and the JSON stat page does not seem to have been much updated either. However, if there is interest in the package, I can look at updating it so it can be installed in current Julia.

**Comment by klpn** [comment]
Added a  `Project.toml`, and could install the package in a fresh Julia 1.8 installation with `pkg> add https://github.com/klpn/JSONStat.jl`.

**Comment by JHuso7** [comment]
Sorry for delayed answer here... I updated Julia to 1.8.5, og managed to install JSONStat package like you described above! I also finally managed to make the code work as well when getting data from Statistics Norway API and using this package for the json result from this API. So this issue has been resolved perfectly!

About the interest for this package, I know that the Python package pyjstat is frequently used, which JSON-stat themself are also refering to on their webpage. So i guess this package as well would be used more frequently in the future perhaps. I see that you have tested on OECD data, and I have testet on Statistics Norway data. I don't know how it will be for Statistics Sweden og Statistics Denmark thou.. But I wouldnt be surprised if these agencies has many similarities.
